src/index.js

```diff
@@ -150,13 +150,26 @@ async function cli () {
     } else {
       output = input ? (`${input}.svg`) : 'out.svg'
     }
-  }
-  if (!/\.(?:svg|png|pdf|md|markdown)$/.test(output)) {
+  } else if (output === '-') {
+    // `--output -` means write to stdout.
+    output = '/dev/stdout'
+    quiet = true
+
+    if (!outputFormat) {
+      outputFormat = 'svg'
+      warn('No output format specified, using svg. ' +
+        'If you want to specify an output format and supress this warning, ' +
+        'please use `-e <format>.` '
+      )
+    }
+  } else if (!/\.(?:svg|png|pdf|md|markdown)$/.test(output)) {
     error('Output file must end with ".md"/".markdown", ".svg", ".png" or ".pdf"')
   }
-  const outputDir = path.dirname(output)
-  if (!fs.existsSync(outputDir)) {
-    error(`Output directory "${outputDir}/" doesn't exist`)
+  if (output) {
+    const outputDir = path.dirname(output)
+    if (!fs.existsSync(outputDir)) {
+      error(`Output directory "${outputDir}/" doesn't exist`)
+    }
```

Comment:
I don't think we need this `if (output)`, since it looks like `output` is always truthy, as it's set by the previous code.

```suggestion
  const outputDir = path.dirname(output)
  if (!fs.existsSync(outputDir)) {
    error(`Output directory "${outputDir}/" doesn't exist`)
```

Reply:
Noted.  Good catch.